Person.rb

```diff
@@ -20,4 +27,16 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission = true
   end
+
+  def correct_name
+    @name
+  end
 end
+
+
+person = Person.new(22, 'maximilianus')
+person.correct_name
+capitalized_person = CapitalizeDecorator.new(person)
+capitalized_person.correct_name
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+capitalized_trimmed_person.correct_name
```

Comment:
- [ ] Kindly make to check your app  you make sure that you arrange the  propriety as you did when you were initialized you must start by name and then age ' ( 'maximilianus', 22 ) '  also make sure that you add `p or put ` to see the result in the terminal :+1: